src/styles/globals.css

```
@@ -108,6 +120,62 @@
+  }
+
+  & > :last-child {
+    @apply mb-0;
+  }
+
+  :where(p):not(:where([class~="not-prose"], [class~="not-prose"] *)) {
+    @apply mt-3 mb-3;
+  }
+}
+
+@utility extend-touch-target {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)